src/openrct2/object/ObjectFactory.cpp

```diff
@@ -129,7 +129,7 @@ class ReadObjectContext : public IReadObjectContext
         return {};
     }
 
-    void LogWarning(uint32 code, const utf8 * text) override
+    void LogWarning([[maybe_unused]] uint32 code, const utf8* text) override
```

Comment:
This is the only implementation and `code` is not used anywhere here. This argument should be dropped or used. Same for `LogError`.